plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/StringPathUtil.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.deltalake;
+
+import java.io.File;
+import java.util.regex.Pattern;
+
+public final class StringPathUtil
+{
+    public static final Pattern DOUBLE_SLASH_TO_REPLEACE = Pattern.compile("(?<!\\w+:/?)//+");
+
+    private StringPathUtil() {}
+
+    public static String stringPathOf(String... pathParts)
+    {
+        String path = String.join(File.separator, pathParts);
```

Comment:
`File.separator` is operating system dependent, but I don't think we want that. Either use / or define a PATH_SEPARATOR constant on this class.
```suggestion
        String path = String.join("/", pathParts);
```